build/linux/installer/scripts/dcr-config-parser.rb

```diff
@@ -21,46 +21,41 @@
 return if !@os_type.nil? && !@os_type.empty? && @os_type.strip.casecmp('windows').zero?
 return unless ENV['USING_AAD_MSI_AUTH']&.strip&.casecmp?('true')
 
```

Comment:
could you share the test results here and that data won't be collected when logs and events profile is enabled